src/sentry/static/sentry/app/views/onboarding/createProject.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import {browserHistory, Link} from 'react-router';
+
+import ApiMixin from '../../mixins/apiMixin';
+import ProjectActions from '../../actions/projectActions';
+import {getPlatformName} from './utils';
+
+import OnboardingProject from '../onboarding/project';
+import {t} from '../../locale';
+
+import Raven from 'raven-js';
+
+const CreateProject = React.createClass({
+  propTypes: {
+    getDocsUrl: React.PropTypes.func
+  },
+
+  contextTypes: {
+    organization: React.PropTypes.object,
+    location: React.PropTypes.object
```

Comment:
Doesn't react-router provide this as a prop?

Reply:
https://github.com/getsentry/sentry/blob/new-project-flow/src/sentry/static/sentry/app/views/app.jsx#L28

you might be right - it's also used as context all over though ;/

Reply:
if you grep for `this.props.location` it seems to be used more than `this.context.location`